store/productsDraft/mutations.js

```diff
@@ -2,53 +2,65 @@
  * Copyright © Bold Brand Commerce Sp. z o.o. All rights reserved.
  * See LICENSE for license details.
  */
+export const types = {
+    SET_PRODUCT_ID: 'SET_PRODUCT_ID',
+    SET_PRODUCT_SKU: 'SET_PRODUCT_SKU',
+    SET_PRODUCT_TEMPLATE: 'SET_PRODUCT_TEMPLATE',
+    SET_PRODUCT_CATEGORIES: 'SET_PRODUCT_CATEGORIES',
+    SET_DRAFT_LANGUAGE_CODE: 'SET_DRAFT_LANGUAGE_CODE',
+    SET_TEMPLATES: 'SET_TEMPLATES',
+    SET_CATEGORIES: 'SET_CATEGORIES',
+    SET_PRODUCT_COMPLETENESS: 'SET_PRODUCT_COMPLETENESS',
+    SET_LAYOUT_ELEMENTS: 'SET_LAYOUT_ELEMENTS',
+    SET_PRODUCT_DRAFT: 'SET_PRODUCT_DRAFT',
+    SET_PRODUCT_TEMPLATE_ELEMENT_VALUE: 'SET_PRODUCT_TEMPLATE_ELEMENT_VALUE',
+    CLEAR_STATE: 'CLEAR_STATE',
+};
+
 export default {
-    setProductId: (state, id) => {
+    [types.SET_PRODUCT_ID](state, id) {
         state.id = id;
     },
-    setProductSku: (state, sku) => {
+    [types.SET_PRODUCT_SKU](state, sku) {
         state.sku = sku;
     },
-    setProductTemplate: (state, template) => {
+    [types.SET_PRODUCT_TEMPLATE](state, template) {
         state.template = template;
     },
-    setProductCategories: (state, selectedCategories) => {
+    [types.SET_PRODUCT_CATEGORIES](state, selectedCategories) {
         state.selectedCategories = selectedCategories;
     },
-    clearStorage: (state) => {
-        state.draft = {};
-        state.id = null;
-        state.sku = '';
-        state.template = '';
-        state.selectedCategories = [];
-        state.languageCode = '';
-        state.layoutElements = [];
-        state.completeness = {};
-        state.templates = [];
-        state.categories = [];
-    },
-    setDraftLanguageCode: (state, languageCode) => {
+    [types.SET_DRAFT_LANGUAGE_CODE](state, languageCode) {
         state.languageCode = languageCode;
     },
-    setTemplates: (state, templates) => {
+    [types.SET_TEMPLATES](state, templates) {
         state.templates = templates;
     },
-    setCategories: (state, categories) => {
+    [types.SET_CATEGORIES](state, categories) {
         state.categories = categories;
     },
-    setProductCompleteness: (state, completeness) => {
+    [types.SET_PRODUCT_COMPLETENESS](state, completeness) {
         state.completeness = completeness;
     },
-    setLayoutElements: (state, layoutElements) => {
+    [types.SET_LAYOUT_ELEMENTS](state, layoutElements) {
         state.layoutElements = layoutElements;
     },
-    setProductDraft: (state, draft) => {
+    [types.SET_PRODUCT_DRAFT](state, draft) {
         state.draft = draft;
     },
-    setProductTemplateElementValue: (state, { index, value }) => {
+    [types.SET_PRODUCT_TEMPLATE_ELEMENT_VALUE](state, { index, value }) {
         state.layoutElements[index].value = value;
     },
-    setProductsValueForKey: (state, { key, value }) => {
-        state.product[key] = value;
+    [types.CLEAR_STATE](state) {
```

Comment:
maybe it's better way : 
`
import defaultState from './state';
[types.CLEAR_STORAGE](state) {
        const states = defaultState();
        Object.keys(states).forEach((key) => {
            state[key] = states[key];
        });
    },
`